src/arrayMethodSort.js

```diff
@@ -4,9 +4,22 @@
  * Implement method Sort
  */
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
-  };
+  [].__proto__.sort2
+    = function(compareFunction = (a, b) => a.toString() > b.toString()) {
+      chooseMethod(this, compareFunction);
+
+      return this;
+
+      function chooseMethod(array, method) {
```

Comment:
1. move default parameter outside applyCustomSort
2. no needs to declare chooseMethod function. You can just use its solution